whois-query/src/main/java/net/ripe/db/whois/query/query/SearchKey.java

```diff
@@ -6,10 +6,13 @@
 import net.ripe.db.whois.common.iptree.Ipv6RouteEntry;
 import net.ripe.db.whois.common.rpsl.attrs.AsBlockRange;
 import net.ripe.db.whois.common.rpsl.attrs.AttributeParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.regex.Pattern;
 
 class SearchKey {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SearchKey.class);
```

Comment:
These error() statements are going to generate a lot of exceptions on bad user input, can they be ignored or log at debug